fdbserver/storageserver.actor.cpp

```diff
@@ -7575,8 +7589,9 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 				// e.g we don't want to copy from blob any more when it's applying mutation logs(APPLYING_MLOGS)
 				if (rangeStatus.second.phase == BlobRestorePhase::COPYING_DATA ||
 				    rangeStatus.second.phase == BlobRestorePhase::ERROR) {
+					wait(loadBGTenantMap(&data->tenantData, &tr));
```

Comment:
Did you get a chance to implement the optimization we talked about here? Or did you want to do that in a separate PR since this is passing correctness?